Index: kde-common/admin/acinclude.m4.in
===================================================================
RCS file: /home/kde/kde-common/admin/acinclude.m4.in,v
retrieving revision 2.455
diff -u -p -r2.455 acinclude.m4.in
--- kde-common/admin/acinclude.m4.in	12 May 2004 08:09:31 -0000	2.455
+++ kde-common/admin/acinclude.m4.in	3 Jun 2004 10:52:36 -0000
@@ -428,7 +428,7 @@ AC_DEFUN([KDE_SET_DEFAULT_PATHS],
 if test "$1" = "default"; then
 
   if test -z "$kde_htmldir"; then
-    kde_htmldir='\${datadir}/doc/HTML'
+    kde_htmldir='/usr/share/doc/kde/HTML'
   fi
   if test -z "$kde_appsdir"; then
     kde_appsdir='\${datadir}/applnk'
@@ -4276,32 +4276,11 @@ AC_DEFUN([AC_CHECK_UTMP_FILE],
 [
     AC_MSG_CHECKING([for utmp file])
 
-    AC_CACHE_VAL(kde_cv_utmp_file,
-    [
-    kde_cv_utmp_file=no
-
-    for ac_file in    \
-                      \
-	/var/run/utmp \
-	/var/adm/utmp \
-	/etc/utmp     \
-     ; \
-    do
-     if test -r "$ac_file"; then
-       kde_cv_utmp_file=$ac_file
-       break
-     fi
-    done
-    ])
+    AC_CACHE_VAL(kde_cv_utmp_file, [kde_cv_utmp_file=/var/run/utmpx])
 
-    if test "$kde_cv_utmp_file" != "no"; then
-	AC_DEFINE_UNQUOTED(UTMP, "$kde_cv_utmp_file", [Define the file for utmp entries])
-	$1
-	AC_MSG_RESULT($kde_cv_utmp_file)
-    else
-    	$2
-	AC_MSG_RESULT([non found])
-    fi
+    AC_DEFINE_UNQUOTED(UTMP, "$kde_cv_utmp_file", [Define the file for utmp entries])
+    $1
+    AC_MSG_RESULT(We are using $kde_cv_utmp_file in PLD instead.)
 ])
 
 
@@ -5078,7 +5057,7 @@ if test "${with_qt_dir+set}" = set; then
   kde_qtdir="$with_qt_dir"
 fi
 
-AC_FIND_FILE(qsql.html, [ $kde_qtdir/doc/html $QTDIR/doc/html /usr/share/doc/packages/qt3/html /usr/lib/qt/doc /usr/lib/qt3/doc /usr/lib/qt3/doc/html /usr/doc/qt3/html /usr/doc/qt3 /usr/share/doc/qt3-doc /usr/share/qt3/doc/html /usr/X11R6/share/doc/qt/html ], QTDOCDIR)
+AC_FIND_FILE(qsql.html, [ /usr/share/doc/qt-doc/html ], QTDOCDIR)
 AC_MSG_RESULT($QTDOCDIR)
 
 AC_SUBST(QTDOCDIR)
