--- kopete_skype/configure.in.in~	2007-01-11 02:59:46.000000000 +0200
+++ kopete_skype/configure.in.in	2007-01-11 03:16:17.000000000 +0200
@@ -11,7 +11,7 @@
 search_incs="$kde_includes $kde_extra_includes /usr/include /usr/include/dbus-1.0 /usr/local/include /usr/local/include/dbus-1.0"
 #look for the connection qt header, it should be there (I'm not interested in the low-level dbus.h)
 AC_FIND_FILE(dbus/dbus.h, $search_incs, dbus_incdir)
-search_incs_arch_deps="$kde_includes $kde_extra_libs/dbus-1.0/include /usr/lib/dbus-1.0/include /usr/local/lib/dbus-1.0/include"
+search_incs_arch_deps="$kde_includes $kde_extra_libs/dbus-1.0/include $kde_libraries/dbus-1.0/include $kde_extra_libs/dbus-1.0/include /usr/lib/dbus-1.0/include /usr/local/lib/dbus-1.0/include"
 #look for the arch-deps header
 AC_FIND_FILE(dbus/dbus-arch-deps.h, $search_incs_arch_deps, dbus_incdir_arch_deps)
 #was it found?
